test/api/publish.test.js

```diff
@@ -152,3 +157,77 @@ describe('publishing endpoints', () => {
     );
   });
 });
+
+describe('API versioning', () => {
+  function getPreconfiguredRequest() {
+    // In these tests we'll generate errors, which triggers a lot of output from
+    // the log. Let's silence that.
+    jest.spyOn(process.stdout, 'write').mockImplementation(() => {});
```

Comment:
Should we snapshot this output? In the future if this breaks, it might be confusing if we have no log information.

Reply:
Ah good idea, I'll look into that.

Reply:
Ah good idea, I'll look into that.